spec/forms/prior_authority/steps/ufn_form_spec.rb

```diff
@@ -25,7 +25,7 @@
       it 'has a validation error on the field' do
         expect(form).not_to be_valid
         expect(form.errors.of_kind?(:ufn, :blank)).to be(true)
-        expect(form.errors.messages[:ufn]).to include('Enter your unique file number for this application')
+        expect(form.errors.messages[:ufn]).to include('Enter the unique reference number')
```

Comment:
i would  question some of these changes. this one in particular struck me as odd when ii saw it on the spreadsheet as it is referring to the UFN using two differently terminologies, which seems unnecessarily confusing.

Reply:
Good point! I've checked with Chris and he confirmed we should standardise on 'file' instead of 'reference'. I've updated.